src/tools/http_upload.c

```diff
@@ -0,0 +1,339 @@
+/*
+ * http_upload.c
+ *
+ * Copyright (C) 2016 Dominik Heidler <dominik@heidler.eu>
```

Comment:
Would you be ok to change the copyright headers to be the same as other files (i.e. under my name)?